resources/projects/jboss-eap/pom.xml

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Would you mind use this quickstart for EAP: https://github.com/jboss-developer/jboss-eap-quickstarts/blob/8.0.x/helloworld/pom.xml
It would be simpler.

Reply:
Done